Farmland.py

```diff
@@ -1,205 +0,0 @@
-
-
-            #this list will hold the points from the last step added as we search through this array
-            last_generation =  [firstpoint]
-            #this list will hold the new points we find
-            next_generation = []
-
-            while len(last_generation) != 0: #When we have counted every point last_generation will be empty
-                for point in last_generation:
-                    #Case 1, checking to see if South of point is fertile and unaccounted for
-                    if point[1] > 0:     #check to see if the point is on the edge
-                        if self.fertilePoints[point[0],point[1]-1] == 1 and Alreadycounted[point[0],point[1]-1] ==0:
-                            next_generation.append([point[0],point[1]-1])
-                            Alreadycounted[point[0],point[1]-1] = number_of_plot
-                            AlreadycountedArea = AlreadycountedArea + 1
-
-                    #Case 2, checking if West of point is fertile and unaccounted for
-                    if point[0]>0: #check to see if the point is on the edge
-                        if self.fertilePoints[point[0]-1,point[1]] == 1 and Alreadycounted[point[0]-1,point[1]] ==0:
-                            next_generation.append([point[0]-1,point[1]])
-                            Alreadycounted[point[0]-1,point[1]] = number_of_plot
-                            AlreadycountedArea = AlreadycountedArea + 1
-
-                    #Case 3, checking if South of point is fertileland and unaccounted for
-                    if point[1] < self.height-1: #check to see if the point is on the edge
-                        if self.fertilePoints[point[0],point[1]+1] == 1 and Alreadycounted[point[0],point[1]+1] ==0:
-                            next_generation.append([point[0],point[1]+1])
-                            Alreadycounted[point[0],point[1]+1] = number_of_plot
-                            AlreadycountedArea = AlreadycountedArea + 1
-
-                    #Case 4, checking if East of point is fertile and unaccounted for
-                    if point[0] < self.width-1: #check to see if the point is on the edge
-                        if self.fertilePoints[point[0]+1,point[1]] == 1 and Alreadycounted[point[0]+1,point[1]] ==0:
-                            next_generation.append([point[0]+1,point[1]])
-                            Alreadycounted[point[0]+1,point[1]] = number_of_plot
-                            AlreadycountedArea = AlreadycountedArea + 1
-
-                #Now we take the new points we found and set them as last_generation
-                #These are points that will be explored in the next step
-                last_generation = next_generation
-                next_generation = []
-
-        #initializes the class attributes
-        self.number_of_Plots = number_of_plot
-        self.plots_of_Land = Alreadycounted
-        return
-
-#given your list of vertices
-def Calculate_Areas(vertices_list):

```

Comment:
camelCase

Reply:
When do I use PascalCase? should I change all variable and method names to camelCase?